test/parallel/test-webcrypto-randomuuid.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const {
+  webcrypto: { randomUUID }
+} = require('crypto');
+
+const last = [
+  '00000000-0000-0000-0000-000000000000'
+];
+
+// Generate a number of UUID's to make sure we're
+// not just generating the same value over and over
+// and to make sure the batching changes the random
+// bytes.
+for (let n = 0; n < 100; n++) {
+  const uuid = randomUUID();
+  assert(!last.includes(uuid));
```

Comment:
Nit: could be a `Set` instead.

Reply:
Could be but given that it's just a simple test the array includes works just fine here.